src/formpack/utils/spss.py

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
+# 😬
+
+from __future__ import unicode_literals
+from collections import OrderedDict
+
+# Constants
+LINE_LIMIT_BYTES = 251
+VARIABLE_LABEL_LIMIT_BYTES = 255
+VALUE_LABEL_LIMIT_BYTES = 120
+SELECT_MULTIPLE_NAME_DELIMITER = '_'
+SELECT_MULTIPLE_LABEL_DELIMITER = ' :: '
+''' SPSS limits variable names to 64 bytes, but we'll create more problems for
+people if we try to change variable names here. Any change to them would
+require altering data exports as well; otherwise, the labels we generate would
+be useless '''
+VARIABLE_NAME_LIMIT_BYTES = 64
+
+
+# HELPERS --------------------------
+
+# https://stackoverflow.com/questions/30686701/python-get-size-of-string-in-bytes
+def utf8_len(s):
+    return len(s.encode('utf-8'))
+
+def spss_escape(s):
+    return s.replace("'", "''").replace("\n", "\\n")
+
+def utf8_ellipsize(s, max_bytes, ellipsis='…'):
+    r"""
+    If necessary, truncate the string `s` and concatenate the result with
+    `ellipsis` such that the final string length does not exceed `max_bytes`
+    :Example:
+        >>> utf8_ellipsize('These 🐿🐿 have taken too many bytes!',
```

Comment:
This example may raise an error in Python2 because of the type of `'These 🐿🐿 have taken too many bytes!'` which is `str`, not `unicode`. 

This makes me think `s` should be an `unicode` before being passed to `utf8_len` otherwise it may raise an exception if it contains non ASCII characters.

Reply:
Got that new hotness: `from __future__ import unicode_literals` ;)